### ISSUE TITLE
Does `LS_COLORS` should as same as `LSCOLORS`? 

### ISSUE THREAD

**Issue by ccqpein** [ohmyzsh/ohmyzsh#11554]
### Describe the bug

After updated the oh my zsh days ago, I notice the color of `cd`'s autocompletion changed. I have a terminal session that still the old color, after several search and test. I find my `used to color` terminal is `LSCOLORS="Gxfxcxdxbxegedabagacad"` and the `LS_COLORS` is unset. 

The new environment after omz updated is `LSCOLORS=Gxfxcxdxbxegedabagacad` and `LS_COLORS="di=34:ln=35:so=32:pi=33:ex=31:bd=34;46:cd=34;43:su=37;41:sg=30;43:tw=30;42:ow=34;42:"`.

After check GitHub commit history, this [commit]( https://github.com/ohmyzsh/ohmyzsh/commit/95d0c4b603e0c880bcf20bc9211b2162e94ed925) set the `LS_COLORS`. 

However, depending this [website](https://geoff.greer.fm/lscolors/) and I tested in my local, the `Gxfxcxdxbxegedabagacad` should be `di=1;36:ln=35:so=32:pi=33:ex=31:bd=34;46:cd=34;43:su=30;41:sg=30;46:tw=30;42:ow=30;43`. 

Should it be sync to each other? 

### Steps to reproduce

1. Add `unset LS_COLORS` in `~/.zshrc` and `zstyle ':completion:*' list-colors "${(@s.:.)LS_COLORS}"` make my `used color` back
2. Or `export LS_COLORS="di=1;36:ln=35:so=32:pi=33:ex=31:bd=34;46:cd=34;43:su=30;41:sg=30;46:tw=30;42:ow=30;43"` and `zstyle ':completion:*' list-colors "${(@s.:.)LS_COLORS}"` can sync the directories colors between `cd` autocompletion and the `ls`.

### Expected behavior

1. `LS_COLORS` and `LSCOLORS` are sync
2. Or doesn't change my color. 

### OS / Linux distribution

macOS 13.2.1

### Zsh version

5.9

### Oh My Zsh version

master (92387d9)

### Terminal emulator

iTerm2

### If using WSL on Windows, which version of WSL

None

**Comment by ccqpein** [comment]
Also, in the commit I post. There is a function `omz_set_completion_colors`, but it isn't there in master branch. Is it been over written accidentally?

**Comment by mcornella** [comment]
You are correct that https://geoff.greer.fm/lscolors/ reports that the same value is `di=1;36:ln=35:so=32:pi=33:ex=31:bd=34;46:cd=34;43:su=30;41:sg=30;46:tw=30;42:ow=30;43`.

I first used that site and found the old value to be equivalent for some reason. I have fixed that and pushed it to master.

> ~~Also, in the commit I post. There is a function `omz_set_completion_colors`, but it isn't there in master branch. Is it been over written accidentally ~~

This was later changed in d342b353 because it was forcibly overwriting user settings.


Thank you for the report!
